src/constants/constant.ts

```diff
@@ -0,0 +1,21 @@
+export const GOV_TOKEN_ASSET = "gov-token";
```

Comment:
It seems like there are a few constants in this file which we are not using. Please remove not used constants.